### DISCUSSION TITLE
How do I pass the --report-trx command line arg?

### DISCUSSION THREAD

**Discussion by macfarmw** [thomhurst/TUnit#976] [Q&A] [answered]
I'm getting stuck on the command line execution of the tests.

```
dotnet --version
8.0.403
```
If I cd to the test project directory and run this
`dotnet run`
It works fine and reports the results to the console

If I try
`dotnet run --report-trx --results-directory "/results"`
I get
```
    TUnit v0.1.1051.0 | 64-bit | Microsoft Windows 10.0.22631 | win-x64 | .NET 8.0.10 | Microsoft Testing Platform v1.4.1

Unknown option '--report-trx'
Usage DecisionTrust.Api.UnitTests.TUnit.exe [option providers] [extension option providers]
...
```
I also tried
`dotnet run --results-directory "/results"`
I don't get any errors, but no test results file appears in the results folder trx or otherwise.

Am I missing a dependency maybe?

**Comment by thomhurst** [comment, marked as answer]
Yeah you need the trx extension package. Details here: https://thomhurst.github.io/TUnit/docs/extensions/

**Reply by macfarmw** [reply]
That fixed it. Thanks!
